Doc/library/test.rst

```diff
@@ -356,6 +356,11 @@ The :mod:`test.support` module defines the following constants:
 
    Check for presence of docstrings.
 
+.. data:: TEST_HTTP_URL
+
+   Define the URL of a dedicated HTTP server for the network tests
+   (http://www.pythontest.net).
```

Comment:
Please remove the value of the constant, it must not be documented. The value doesn't matter, and it reduces the maintenance burden to not have to update the doc, each time the URL changes.